ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/IgnoreUnusedBindingExpressionFix.fs

```diff
@@ -0,0 +1,54 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Util
+open JetBrains.ReSharper.Psi
+open JetBrains.ReSharper.Psi.ExtensionsAPI.Tree
+open JetBrains.ReSharper.Resources.Shell
+
+type IgnoreUnusedBindingExpressionFix(warning: UnusedValueWarning) =
+    inherit FSharpQuickFixBase()
+
+    let pat = warning.Pat.IgnoreParentParens()
+    let binding = BindingNavigator.GetByHeadPattern(pat)
+    let letOrUseExpr = LetOrUseExprNavigator.GetByBinding(binding)
+    
+    let ignoreInnermostExpression (expr: IFSharpExpression) =
+        let rec getInnermostExpression (expr: IFSharpExpression) =
+            match expr with
+            | :? ISequentialExpr as seqExpr -> getInnermostExpression (seqExpr.Expressions.Last())
```

Comment:
`ExpressionsEnumerable` is a bit preferable since it doesn't store the whole collection.